lib/inspec/base_cli.rb

```diff
@@ -60,6 +60,8 @@ def self.exec_options
         desc: 'Load attributes file (experimental)'
       option :cache, type: :string,
         desc: 'Use the given path for caching dependencies. (default: ~/.inspec/cache)'
+      option :no_write_lockfile, type: :boolean, default: false,
```

Comment:
that is leading to a strange cli output:
`[--no-write-lockfile], [--no-no-write-lockfile]  # Do not write out a lockfile based on this execution`
I think we need to switch to `write-lockfile` with default `true`